maa-cli/src/run/copilot.rs

```diff
@@ -0,0 +1,207 @@
+    // Determine type of stage
+    let mut task_type: &str = "Copilot";
+    let content_type = recast["type"].clone();
+    if content_type == "SSS" {
+        task_type = "SSSCopilot";
+    }
+
+    // Print stage info
+    println!("Stage info:\n");
+    if task_type == "Copilot" {
+        let stage_code_name = recast["stage_name"].as_str().unwrap();
+        let stage_name = find_stage_name(stage_code_name, stage_dir);
+        println!("{}\n", stage_name);
+    } else {
+        let stage_name = recast["stage_name"].as_str();
+        println!("{}\n", stage_name.unwrap());
+    }
+
+    //Print operators info
+    println!("Operator lists:\n");
+    let opers = recast["opers"].clone();
+    for operator in opers.as_array().unwrap() {
+        let name = operator["name"].as_str().unwrap();
+        let skill = operator["skill"].as_u64().unwrap();
+        println!("{}, Skill: {};", name, skill,);
+    }
+
+    //Get input of user
+    let mut formation: bool = true;
+    let mut loop_times: i64 = 1;
+    if task_type == "Copilot" {
+        let input: Input<String> = Input::new(
+            Some("yes".to_string()),
+            Some("Whether to quick build(Y/N):"),
+        );
+        let stdin = io::stdin();
+        let stdout = io::stdout();
+        let _ = input.prompt(stdout.lock());
+        let result = input.ask(stdout.lock(), stdin.lock());
+        match result.unwrap().as_str() {
+            "yes" | "Yes" | "y" | "Y" => {
+                formation = true;
+            }
+            "no" | "No" | "n" | "N" => {
+                formation = false;
+            }
+            _ => {
+                println!("Invaild input");
+                return false;
+            }
+        }
+    } else {
+        let input: Input<i64> = Input::new(Some(1), Some("Loop times:"));
+        let stdin = io::stdin();
+        let stdout = io::stdout();
+        let _ = input.prompt(stdout.lock());
+        let result = input.ask(stdout.lock(), stdin.lock());
+        match result {
+            Ok(value) => {
+                loop_times = value;
+            }
+            Err(_error) => {
+                println!("Invaild input");
+                return false;
+            }
+        }
+    }
+
+    // Save json file
+    let json_file_name = format!("/{}.json", copilot_id);
+    let json_path = format!("{}{}", cache_dir, json_file_name);
+    write_file(content, json_path.clone());
+
+    // Append task
+    let t_type = if task_type == "Copilot" {
+        TaskType::Copilot
+    } else {
+        TaskType::SSSCopilot
+    };
+    let mut task_list: Vec<Task> = Vec::new();
+    task_list.push(Task::new(
+        t_type,
+        object!(
+            "filename" => json_path,
+            "formation" => formation,
+            "loop_times" => loop_times,
+        ),
+        Strategy::default(),
+        default_variants(),
+    ));
+
+    let task = TaskList { tasks: task_list };
+    let _ = run(dirs, task, None, false, false, false);
```

Comment:
We can return the result value of `run` here, if this function returns `Result<()>`.